src/sync/serversync.js

```diff
@@ -1,29 +1,17 @@
-﻿import MediaSync from './mediasync';
-import ContentUploader from './contentuploader';
-import localAssetManager from '../localassetmanager';
+﻿import MediaSync from 'sync/mediasync';
+import localAssetManager from 'localassetmanager';
 
 function performSync(connectionManager, server, options) {
 
     console.log(`ServerSync.performSync to server: ${server.Id}`);
 
     options = options || {};
 
-    const cameraUploadServers = options.cameraUploadServers || [];
-    console.log("ServerSync cameraUploadServers: " + JSON.stringify(cameraUploadServers));
-
-    const uploadPhotos = cameraUploadServers.includes(server.Id);
-    console.log("ServerSync uploadPhotos: " + uploadPhotos);
-
-    const promise = uploadPhotos ? uploadContent(connectionManager, server, options) : Promise.resolve();
+    const promise = Promise.reject();
```

Comment:
can we just do smth like `return Promise.reject()` instead of the whole function?
plus a TODO to remove and a deprecation warning?